### PR TITLE
Autocomplete update, view pages updated

### LINKED ISSUE #94: Autocomplete fix
Remove the glimpse of the senate number which flashes before switching to the show page

### LINKED ISSUE #95: Fix twitching in tooltip hoverover function (JS)

### LINKED ISSUE #96: change login message in show page to "Log in to add this representative to your dashboard"

### LINKED ISSUE #97: Autocomplete by last name

### PR DESCRIPTION
**Pull request by Rostammahabadi**
Updated features:
When a user clicks back after selecting a state or representative name in the autocomplete, it clears everything out
When a user clicks on a name in the autocomplete, the name is entered in the box instead of the value

Both senator and house_member view pages updated:
When i hover over the tooltip, the picture no longer moves with the tooltip box

Tests
Updated and all are passing

### CODE REVIEW

File: app/views/house_members/show.html.erb [hunk 1]
```diff
@@ -38,7 +44,7 @@
         <% elsif current_user && !current_user.house_members.include?(@rep)%>
           <%= button_to "Add to Favorites" , "/house_members/#{@rep.id}/favorites", method: :post%>
         <% else %>
-           <p>Click on Register to favorite</p>
+           <p>Log in to add this representative to your dashboard</p>
```
Comment (dborski) [inline app/views/house_members/show.html.erb:47]: I definitely like this wording better, nice fix

**Comment by dborski** [comment]
Nice work!

**Review by dborski — APPROVED** [review]

File: app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -230,4 +230,10 @@
...
 	top:1px;
-}
\ No newline at end of file
+}
+.fixed-image{
+  position:fixed;
+  margin-left:780px;
+  margin-top:100px;
+  width:350px;
+}
```
Comment (AntonioJacksonII) [inline app/assets/stylesheets/application.scss:239]: Yes! It's great the image is fixed now.

File: app/views/house_members/show.html.erb (hunk 1 above)
Comment (AntonioJacksonII) [reply app/views/house_members/show.html.erb:47]: I agree. This wording is much more clear to the user.

**Review by AntonioJacksonII — APPROVED** [review]

**Pull request merged**